examples/zkapps/08-custom-tokens/package.json

```diff
@@ -43,6 +43,6 @@
     "typescript": "^4.7.2"
   },
   "peerDependencies": {
-    "o1js": "0.15.*"
+    "o1js": "1.0.*"
```

Comment:
this is `1.0.*` but others are `0.18.*`?

Reply:
This example has actually been removed from the current tutorials and Mina Foundation is working on a new one. Either way I will update this. Thanks!